src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,14 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ comment }) => (
```

Comment:
Since the comment prop is an object, you can use destructuring to immediately extract name, email, and body from it:
`const { name, email, body } = comment;`
This way, you won't need to repeatedly write `comment.name`,` comment.email`, and `comment.body`.